puterbot/crud.py

```diff
@@ -68,6 +68,46 @@ def insert_window_event(recording_timestamp, event_timestamp, event_data):
     }
     _insert(event_data, WindowEvent, window_events)
 
+def insert_perf_stat(recording_timestamp, event_type, start_time, end_time, buffer=None):
```

Comment:
Why not re-use `_insert` here, like the other events?

Reply:
Changes made with latest commit.

Reply:
Thank you @apgorton ! In the future please include the commit hash in your comment so that the reader can easily click through 🙏